src/changes/changes.xml

```diff
@@ -22,6 +22,7 @@
   <body>
 
   <release version="3.5" date="tba" description="tba">
+    <action issue="LANG-1114" type="fix" dev="acoates">Fix bug in TypeUtils.equals where comparison of wildcard and nom-wildcard type was incorrectly returning true</action>
```

Comment:
Usually we use the dev attribute for the committer who reviews the patch. Contributors are attributed in the due-to attribute. I hope it is okay for you if I modify this when I apply this fix.